app/src/main/AndroidManifest.xml

```diff
@@ -124,8 +124,12 @@
       android:name="net.squanchy.notification.NotificationsIntentService"
       android:exported="false" />
 
+    <service
+      android:name=".notification.NearNotificationIntentService"
+      android:exported="false" />
+
     <receiver
-      android:name="it.near.sdk.recipes.background.NearItBroadcastReceiver"
+      android:name=".notification.NearNotificationReceiver"
```

Comment:
Can you undo this and use the full package name in the one you just added please?